driver-core/src/main/java/com/datastax/driver/core/Connection.java

```diff
@@ -704,9 +660,7 @@ public void shutdown() {
             // we're not on an I/O thread or anything, so just call await.
             allChannels.close().awaitUninterruptibly();
 
-            // This will shut down I/O threads. Since this is called only once all connection have
-            // been individually closed, it's fine.
-            eventLoopGroup.shutdownGracefully().syncUninterruptibly();
+            nettyCustomizer.shutdownEventLoopGroup(eventLoopGroup);
```

Comment:
The event loop instance is provided by NettyCustomizer, so it is up to NettyCustomizer to close it properly.